### PR TITLE
Localize GOG extras

### PR DESCRIPTION
**Pull request by bexnoss**
Instead of localizing by IP this makes it based on the Lutris locale.

### CODE REVIEW

File: lutris/services/gog.py [hunk 1]
```diff
@@ -95,6 +95,20 @@ def __init__(self):
...
+            "fr": "fr-FR",
+            "pl": "pl-PL",
+            "ru": "ru-RU",
+            "zh": "zh-Hans",
+        }
+        language = i18n.get_lang()
+        if language in gog_locales:
+            self.locale = gog_locales[language]
+        else:
+            self.locale = "en-US"
```
Comment (AlexanderRavenheart) [inline lutris/services/gog.py:110]:
```suggestion
        self.locale = gog_locales.get(i18.n.get_lang(), "en-US")
```

Comment (bexnoss) [reply lutris/services/gog.py]: That's much better, thanks!

File: lutris/services/gog.py [hunk 2]
```diff
@@ -95,6 +95,16 @@ def __init__(self):
         super().__init__()
         self.selected_extras = None
 
+        gog_locales = {
```
Comment (AsciiWolf) [inline lutris/services/gog.py:98]: Is it possible to add more locales or are these the only ones supported by GOG extras?

Comment (bexnoss) [reply lutris/services/gog.py:98]: These are all locales supported by GOG.

**Pull request merged**